README.md

```diff
@@ -1,160 +1,132 @@
-"CLP", "CNY", "CZK", "DKK", "EUR", "GBP", "HKD", "HUF", "IDR", "ILS", 
-"INR", "JPY", "KRW", "MXN", "MYR", "NOK", "NZD", "PHP", "PKR", "PLN",
-"RUB", "SEK", "SGD", "THB", "TRY", "TWD", "ZAR", "USD".
-
-The other values should be self-explanatory,
-if not feel free to raise a github issue.
-
-### Prerequisites
-* python3.6
-* sqlite
-* [TA-lib](https://github.com/mrjbq7/ta-lib#dependencies) binaries
-* Minimal (advised) system requirements: 2GB RAM, 1GB data, 2vCPU
-
-### Install
-
-#### Arch Linux
-
-Use your favorite AUR helper and install `python-freqtrade-git`.
-
-#### Manually
-
-`master` branch contains the latest stable release.
-
-`develop` branch has often new features, but might also cause breaking changes. To use it, you are encouraged to join our [slack channel](https://join.slack.com/t/highfrequencybot/shared_invite/enQtMjQ5NTM0OTYzMzY3LWMxYzE3M2MxNDdjMGM3ZTYwNzFjMGIwZGRjNTc3ZGU3MGE3NzdmZGMwNmU3NDM5ZTNmM2Y3NjRiNzk4NmM4OGE).
-
-```
-$ cd freqtrade/
-# copy example config. Dont forget to insert your api keys
-$ cp config.json.example config.json
-$ python -m venv .env
-$ source .env/bin/activate
-$ pip install -r requirements.txt
-$ pip install -e .
-$ ./freqtrade/main.py
+[![Coverage Status](https://coveralls.io/repos/github/gcarq/freqtrade/badge.svg?branch=develop&service=github)](https://coveralls.io/github/gcarq/freqtrade?branch=develop)
+
+
+Simple High frequency trading bot for crypto currencies designed to 
+support multi exchanges and be controlled via Telegram.
+
+![freqtrade](https://raw.githubusercontent.com/gcarq/freqtrade/develop/docs/assets/freqtrade-screenshot.png)
+
+## Disclaimer
+This software is for educational purposes only. Do not risk money which 
+you are afraid to lose. Always start by running a trading bot in Dry-run and do not engage money
+before you understand how it works and what profit/loss you should
+expect.
+
+We strongly recommend you to have coding and Python knowledge. Do not 
+hesitate to read the source code and understand the mechanism of this bot.
+
+## Table of Contents
+- [Features](#features)
+- [Quick start](#quick-start)
+- [Documentations](https://github.com/gcarq/freqtrade/blob/develop/docs/index.md)
+   - [Installation](https://github.com/gcarq/freqtrade/blob/develop/docs/installation.md)
+- [Support](#support)
+   - [Help](#help--slack)
+   - [Bugs](#bugs--issues)
+   - [Feature Requests](#feature-requests)
+   - [Pull Requests](#pull-requests)
+- [Basic Usage](#basic-usage)
+  - [Bot commands](#bot-commands)
+  - [Telegram RPC commands](#telegram-rpc-commands)
+- [Min hardware required](#min-hardware-required)
+
+## Branches
+The project is currently setup in two main branches:
+- `develop` - This branch has often new features, but might also cause 
+breaking changes.
+- `master` - This branch contains the latest stable release. The bot 
+'should' be stable on this branch, and is generally well tested. 
+
+## Features
+- [x] **Based on Python**: For botting on any operating system - 
```

Comment:
IIRC we require python3.6+ which should be mentioned somewhere

Reply:
I have added a whole section about the software requirements in the README.md